users/views.py

```diff
@@ -13,7 +13,14 @@
 from rest_framework.permissions import AllowAny
 from rest_framework.throttling import AnonRateThrottle
 from rest_framework_simplejwt.views import TokenObtainPairView
+from .serializers import CustomTokenObtainPairSerializer
+import requests
 
+from django.contrib.auth import get_user_model
```

Comment:
Move this import closer to the model definition for clarity and to follow Django conventions.